packages/functional-tests/playwright.config.ts

```diff
@@ -15,7 +15,7 @@ const CI = !!process.env.CI;
 // see .vscode/launch.json
 const DEBUG = !!process.env.DEBUG;
 const SLOWMO = parseInt(process.env.PLAYWRIGHT_SLOWMO || '0');
-const NUM_WORKERS = parseInt(process.env.PLAYWRIGHT_WORKERS || '16');
+const NUM_WORKERS = parseInt(process.env.PLAYWRIGHT_WORKERS || '4');
```

Comment:
Why are we changing the workers from 16 to 4?

Reply:
@ashrivastava-qa Does 16 work for you?

> Updates the local playwright test workers to 4 because 16 was causing timeout issues (I can't imagine a local machine that can handle 16 playwright test workers at once)

Reply:
Actually @vpomerleau  and I did some testing on the number of workers a while back to make sure if decreasing or increasing the workers has any impact on the test pass/fail rate and while we couldn't derive a clear correlation, 16 worked for both of us. But I also agree with your comment. Let's try and switch it to 4 and monitor for the next few days on how it behaves.

Reply:
I'm running the tests locally with 4 workers out of curiosity - and while it does seem to help when running a subset of tests, I see more timeouts when running the whole suite.

Reply:
@vbudhram @vpomerleau let's keep an eye out and monitor this. In case we start to see more failures then we will switch it back.

Reply:
I think this should be set to 2 to match CI and devs can up it locally to match the power of their locals. I would guess that timeouts and failures that happen due to changing this value are due to the change in run order that occurs when the value is changed.